platform-plugins.md

```diff
@@ -4,267 +4,6 @@ title: Plugins&#58; Integrating with platform-specific code
 permalink: /platform-plugins/
 ---
 
-This guide describes how Flutter apps can integrate with platform-specific code
-available on iOS and Android devices. This includes both device APIs (like
-[`url_launcher`](https://pub.dartlang.org/packages/url_launcher) and
-[`battery`](https://pub.dartlang.org/packages/battery)) and third-party platform SDKs 
-(like [Firebase](https://github.com/flutter/plugins/blob/master/FlutterFire.md)).
+<meta http-equiv="Refresh" content="0; url=/using-packages/" />
```

Comment:
Let's use the Firebase rules for redirects, so we don't need to keep this page around.

Check out https://github.com/flutter/website/blob/master/firebase.json

There's a few redirects in there now.